gulpfile.js

```diff
@@ -0,0 +1,122 @@
+// Copyright 2015 The Oppia Authors. All Rights Reserved.
```

Comment:
Do you run into any issues with shutting down the server? I started up a dev server successfully and then typed Ctrl-C into the terminal. It never terminated; I just kept getting things like this:

 ^C[13:53:18] [gulp-gae] stopping script

Eventually I had to force-close the terminal window. This is kind of undesirable; does it happen on your machine too?